### DISCUSSION TITLE
Do I Have to open 8447 on farmer's machine for remote harvester?

### DISCUSSION THREAD

**Discussion by sergixc** [Chia-Network/chia-blockchain#6010] [Support]
So the question is:
Do I Have to open 8447 on farmer's machine for remote harvester?

**Comment by BrandtH22** [comment]
@sergixc , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - this is dependent on your OS, I know that on some linux distros you have to activate the firewall and open the necessary ports.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
